src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowRunRestartTest.java

```diff
@@ -99,7 +99,6 @@
             assertTrue(p.isResumeBlocked());
             WorkflowRun b = p.getBuildByNumber(1);
             r.waitForCompletion(b);
-            assertFalse(b.executionLoaded);
```

Comment:
Should we invert the assertion instead of deleting it? Would pulling in https://github.com/jenkinsci/jenkins-test-harness/pull/210 make the behavior consistent?

Reply:
> Should we invert the assertion

Not sure I see the point.

> Would pulling in jenkinsci/jenkins-test-harness#210 make the behavior consistent?

Not sure.